app/services/request_referer_service.rb

```diff
@@ -23,6 +23,8 @@ def self.access_whitelisted?(request, controller_name, action_name)
     'host'                     => %w(show:id show_list), # http://localhost:3000/host/show/400r6
     'miq_request'              => %w(show:id show_list), # http://localhost:3000/miq_request/show/400r3
     'ems_cluster'              => %w(show:id show_list), # http://localhost:3000/ems_cluster/show/400r2
+    'ems_cloud'                => %w(show:id show_list),
+    'ems_infra'                => %w(show:id show_list),
```

Comment:
Sure, that is the whitelist for ;-) But do we need to directly access `show#id`?

Or is this added just for the specs to pass (that would be wrong in my opinion).